apps/okreads-e2e/src/integration/reading-list.spec.ts

```diff
@@ -1,6 +1,8 @@
 describe('When: I use the reading list feature', () => {
+  let readingItemsCount = 0;
```

Comment:
why set this value to 0? you're just defining the variable, not setting its value